README.md

```diff
@@ -80,6 +84,39 @@ For more info on providers, please refer to [this](https://github.com/pystardust
 
 ### Linux
 
+### Debian
+
+```
+wget -qO- https://Wiener234.github.io/ani-cli-ppa/KEY.gpg | sudo tee /etc/apt/trusted.gpg.d/ani-cli.asc
+wget -qO- https://Wiener234.github.io/ani-cli-ppa/ani-cli-debian.list | sudo tee /etc/apt/sources.list.d/ani-cli-debian.list
+sudo apt update
+```
+
+### Fedora
+
+To install mpv (and vlc) you need _RPM Fusion free_ enabled. Simply follow the instructions here: https://rpmfusion.org/Configuration
+To be able to install syncplay, you'll need to enable this copr repo: https://copr.fedorainfracloud.org/coprs/batmanfeynman/syncplay/
+
+There are currently two packages for Fedora: `ani-cli` which is a minimal installation that can only use mpv and `ani-cli-full` which contains all the functionality (note that vlc and syncplay are still optional dependencies).
```

Comment:
Why/how?

Reply:
We no longer have two packages, only one that has it all.